source/partials/_footer.html.erb

```diff
@@ -0,0 +1,38 @@
+<footer class="footer">
+  <div class="footer__content">
+
+    <div class="footer__column footer__column--insiter">
```

Comment:
Maybe later, we could change the modifier `--insiter` for a more generic, like `--about`